### ISSUE TITLE
Display error if the app doesn't start

### ISSUE THREAD

**Issue by MattiSG** [openfisca/openfisca-ops#111]
If I already have a process running on the port set up for the app (default `8000`), the app won't start as it cannot bind to that port.

However, Ansible will output:

```
TASK [openfisca_api_fr : Enable and start the systemd service] *****************
changed: [openfisca_api_fr]

TASK [openfisca_api_fr : Check that OpenFisca Web API is actually started] *****
ok: [openfisca_api_fr]
```

The `systemctl` log will show that the service is `inactive (dead)`.

This error should be shown in the Ansible logs.

**Comment by MattiSG** [comment]
@Ndpnt has this been solved in #126?

**Comment by MattiSG** [comment]
Checked today by manually starting a `python3 -m http.server 8000 &` on the machine on a deployment of 0f2eb38cc86b0d88d76938928d6a27c72ea05df2, succeeds:

```
TASK [openfisca_api : Check that OpenFisca Web API is actually started] **************************************************************************************************************************************************************************************************************************************************************************************************
FAILED - RETRYING: [127.0.0.1]: Check that OpenFisca Web API is actually started (5 retries left).
FAILED - RETRYING: [127.0.0.1]: Check that OpenFisca Web API is actually started (4 retries left).
FAILED - RETRYING: [127.0.0.1]: Check that OpenFisca Web API is actually started (3 retries left).
FAILED - RETRYING: [127.0.0.1]: Check that OpenFisca Web API is actually started (2 retries left).
FAILED - RETRYING: [127.0.0.1]: Check that OpenFisca Web API is actually started (1 retries left).
fatal: [127.0.0.1]: FAILED! => changed=false 
  attempts: 5
  content: ''
  elapsed: 0
  msg: 'Status code was -1 and not [300]: Connection failure: Remote end closed connection without response'
  redirected: false
  status: -1
  url: http://0.0.0.0:8000
```
